netlogo-core/src/test/fileformat/PlotConverterTest.scala

```diff
@@ -0,0 +1,163 @@
+package org.nlogo.fileformat
+
+import org.nlogo.core.{ Model }
+import org.nlogo.core.{ Pen, Plot, View }
+import org.scalatest.{ FunSuite, DiagrammedAssertions }
+
+class PlotConverterTests extends FunSuite with ConversionHelper with DiagrammedAssertions {
+
+  /*************
+   * Plot Name *
+   *************/
+
+  test("if the plot names are not initialized"){
+    val model = Model(widgets = Seq( View()
+                                   , Plot(display = None, pens = List(Pen(display = "dep", setupCode = "plot e 10")))
+                                   , Plot(display = None, pens = List(Pen(display = "DeP", setupCode = "plot e 10")))
+                                   , Plot(display = None, pens = List(Pen(display = "Dep", setupCode = "plot e 10")))))
```

Comment:
It would be nice (but not critical) if the rest of the test setups were nicely aligned with whitespace like this one is.